internal/shareable/image.go

```diff
@@ -18,14 +18,14 @@ import (
 	"fmt"
 	"image/color"
 	"runtime"
-	"sync"
 
 	"github.com/hajimehoshi/ebiten/v2/internal/affine"
 	"github.com/hajimehoshi/ebiten/v2/internal/driver"
 	"github.com/hajimehoshi/ebiten/v2/internal/graphics"
 	"github.com/hajimehoshi/ebiten/v2/internal/hooks"
 	"github.com/hajimehoshi/ebiten/v2/internal/packing"
 	"github.com/hajimehoshi/ebiten/v2/internal/restorable"
+	"github.com/hajimehoshi/ebiten/v2/internal/sync"
```

Comment:
This package is tricky due to `runtime.SetFinalizer`. Finalizers run in different goroutines. Is there a guarantee that the main goroutines and the finalizer goroutines never race?

Reply:
There is no guarantee, I will switch it back to the stdlib sync package.